File: exercises/beecrowd/1180/1180.go
// https://judge.beecrowd.com/pt/problems/view/1180
package main

import "fmt"

func main() {
	var n int
	fmt.Scan(&n)

	valores := make([]int, n)
	for i := 0; i < n; i++ {
		fmt.Scan(&valores[i])
	}

	menor := valores[0]
	pos := 0
	for i := 1; i < n; i++ {
		if valores[i] < menor {
			menor = valores[i]
			pos = i
		}
	}

	fmt.Printf("Menor valor: %d\n", menor)
	fmt.Printf("Posicao: %d\n", pos)
}

File: exercises/beecrowd/1180/1180.java
// https://judge.beecrowd.com/pt/problems/view/1180
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();
        int[] arr = new int[n];

        int min = Integer.MAX_VALUE;
        int pos = 0;

        for (int i = 0; i < n; i++) {
            arr[i] = sc.nextInt();
            if (arr[i] < min) {
                min = arr[i];
                pos = i;
            }
        }

        System.out.println("Menor valor: " + min);
        System.out.println("Posicao: " + pos);

        sc.close();
    }
}
